flow/core/vehicles.py

```diff
@@ -685,6 +700,26 @@ def get_speed(self, veh_id, error=-1001):
             return [self.get_speed(vehID, error) for vehID in veh_id]
         return self.__sumo_obs.get(veh_id, {}).get(tc.VAR_SPEED, error)
 
+    def get_default_speed(self, veh_id, error=-1001):
+        """Return the deault next speed of the specified vehicle in SUMO.
```

Comment:
Nit: default is misspelled.
Additionally, this comment should make clear that it's the default speed if no control is applied.

Reply:
Solved.